src/reference-designs/tko-on-vsphere-with-tanzu-nsxt.md

```diff
@@ -0,0 +1,446 @@
+![Istio Service Mesh details (example)](img/tko-on-vsphere-with-tanzu-nsxt/image28.png)
+
+![Application Overview - microservices aware out of the box.](img/tko-on-vsphere-with-tanzu-nsxt/image29.png)
+
+![Detailed Application Dashboards - out of the box.](img/tko-on-vsphere-with-tanzu-nsxt/image30.png)
+
+![Automatic Distributed Tracing information between (micro)services.](img/tko-on-vsphere-with-tanzu-nsxt/image31.png)
+
+To view and download integrations with prebuilt dashboards that are available in Wavefront, see [Integrations](https://vmware.wavefront.com/integrations) on the VMware Tanzu Observability site.
+
+## Tanzu Service Mesh (TSM)
+
+VMware Tanzu Service Mesh is an enterprise-class service mesh solution that provides consistent control and security for microservices, end-users, and data across all your clusters and clouds in the most demanding multi-cluster and multi-cloud environments.
+
+### Key Benefits of Tanzu Service Mesh:
+
+- Extends the service mesh capability (discovery, connectivity, control, security, and observability) to users and data.
+- Facilitates the development and management of distributed applications across multiple clusters, multiple clouds, and in hybrid-cloud environments with Global Namespaces, supporting federation across organizational boundaries, technologies, and service meshes.
+- Implements consistent application-layer traffic management and security policies across all your clusters and clouds.
+- Integrates with VMware Tanzu Mission Control, VMware Enterprise PKS, and VMware Tanzu Kubernetes Grid to provide a seamless user experience.
+
+### Global Namespace
+
+With global namespaces in Tanzu Service Mesh, you can easily connect and secure the services in your application across clusters. You can add the services in your application to a global namespace to have them automatically discovered and connected across the clusters.
+
+The global namespace enables users to view the topology of the services or a cluster as a chart. The topology graph can be used to understand service dependencies and determine the health of the services and the performance metrics related to each service.
+
+Tanzu Service Mesh generates a topology graph view dynamically by observing the traffic that is flowing between the services in a global namespace or a cluster. The topology graph visualizes the communication patterns between the services and helps you gain insights into the service dependencies.
+
+### Recommendation for using Tanzu Service Mesh
+
+- Use Tanzu Service Mesh when containerized applications are deployed in multi-cluster/multi-cloud environments. Using Tanzu Service Mesh, you can apply policies such as security and traffic management at the Global Namespace level.
+- For multi-cluster/multi-cloud applications, that have load balancer requirements, use Tanzu Service Mesh in-built integration with AWS Route 53 or NSX Advanced Load Balancer to leverage global load-balancing for containerized applications.
```

Comment:
Replace / with or.

Reply:
changed to "and", as concepts aren't mutually exclusive